xendit-java-lib/src/main/java/com/xendit/model/cardlessCredit/CardlessCredit.java

```diff
@@ -0,0 +1,122 @@
+package com.xendit.model.cardlessCredit;
+
+import com.google.gson.annotations.SerializedName;
+import com.xendit.Xendit;
+import com.xendit.exception.XenditException;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
```

Comment:
why do we need a constructor for this? all the methods are static and exposing a constructor might be confusing.

Reply:
Yes, package name should be lower case. Will change that. and also will remove constructor if required.